packages/meta-harness/scenarios/fixtures/assert-task-state-outcome.py

```
@@ -173,6 +174,77 @@ def assert_decision_recorded(plans_path: Path, *, expected_decision: str) -> Non
+    (`.claude/context/...`), `__pycache__`, etc. already exist as a normal side effect of
+    running Claude Code in this harness, so treating any of them as collateral damage would
+    make the check flake regardless of what the candidate did. When `allowed_new_prefixes` is
+    non-empty (the handoff suite's case, `(".claude/handoffs/",)`), every untracked path must
+    start with one of the given prefixes, or the run fails.
+
+    `cwd` defaults to the process's own working directory (the production oracle container
+    sets `--workdir /workspace` and relies on the process cwd); tests pass an explicit
+    temporary git repo instead.
+    """
+    diff_result = subprocess.run(
+        ["git", "diff", "--name-only", "HEAD"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)